firefox-ios/Client/Frontend/Browser/TopTabDisplayManager.swift

```diff
@@ -711,13 +712,15 @@ extension TopTabDisplayManager: TabManagerDelegate {
 extension TopTabDisplayManager: Notifiable {
     // MARK: - Notifiable protocol
     func handleNotifications(_ notification: Notification) {
```

Comment:
Should this be marked `nonisolated`?
```suggestion
    func nonisolated handleNotifications(_ notification: Notification) {
```